### ISSUE TITLE
Can't export type declarations on buildContext 

### ISSUE THREAD

**Issue by n2taylor** [ericmakesapps/graphql-passport#93]
When trying to build an npm package that imports `graphql-passport`, exporting the type declarations (`declaration: true`) fails on `buildContext` due to the `CommonRequest` interface not being exported.

```
// not exported
interface CommonRequest<UserObjectType extends Express.User>
  extends Pick<Context<UserObjectType>, 'isAuthenticated' | 'isUnauthenticated'> {
  user?: UserObjectType;
}

export interface Context<UserObjectType extends Express.User> {
  isAuthenticated: () => boolean;
  isUnauthenticated: () => boolean;
  getUser: () => UserObjectType;
  authenticate: (strategyName: string, options?: object) => Promise<AuthenticateReturn<UserObjectType>>;
  login: (user: UserObjectType, options?: object) => Promise<void>;
  logout: (options?: object) => Promise<void>;
  res?: express.Response;
  req: CommonRequest<UserObjectType>;  // <-------- local interface used in exported interface
}
```

This gives the following error:

```
import { buildContext } from 'graphql-passport';
...

const getGraphqlContext = (req: Request, res: Response) => {
    const body: { [key: string]: any } = req.body;
    return {
        ...body,
        ...buildContext({ req, res }),
        ...dataloaders,
    };
};
```

```
Exported variable 'getGraphqlContext' has or is using name 'CommonRequest' from external module "/usr/src/app/node_modules/graphql-passport/lib/buildContext" but cannot be named.
```

**Comment by ericbf** [comment]
If I were to export that interface, would that fix your issue?

**Comment by n2taylor** [comment]
Yes, it looks like all the other interfaces in `buildContext` are fully resolvable. Thanks!

**Comment by ericbf** [comment]
This is released in v0.6.8

**Comment by n2taylor** [comment]
It worked! Thank you
